src/lib/timeseries.js

```diff
@@ -125,6 +125,10 @@ export const computeAggregatedTimeseries = (timeseries, makeSections) => {
       serie,
       'features[1].properties.display_name'
     )
+    const coordinates = {
+      startPoint: getStartPlaceCoordinates(timeserie),
+      endPoint: getEndPlaceCoordinates(timeserie)
+    }
```

Comment:
what about already computed/aggregated timeseries?

Reply:
We won't take them into consideration. Which raises again the migration issue, it would be nice to have a unified strategy about it

Reply:
Here it's pretty simple, you can add a special case for this migration inside your service. No need to have something elsewhere. No?

Reply:
Sure, but : 
- What if I have a lot of existing timeseries to migrate? The service might be killed before finishing, so we need to be careful about this, do it by batch, and restart the service if needed. I feel like this is not as simple as "query / update"
- As coachco2 is not deployed as stable yet, I think this is not a big need to be retro-active
- I feel like we have this issue quite often, but no unified way to do it. I would like to avoid having plenty of "one-shot" code, and think of some unified way to do it as simple as possible :)